src/tv2-common/videoSwitchers/atemToTricasterDveConverter.ts

```diff
@@ -0,0 +1,68 @@
+import { TSR } from 'blueprints-integration'
+import { TriCasterDveConverter } from './TriCasterDveConverter'
+
+const ATEM_WIDTH = 32
+const ATEM_HEIGHT = 18
+
+const ATEM_CROP_LEFT_RIGHT_MAX_VALUE = 32000
+const ATEM_CROP_TOP_BOTTOM_MAX_VALUE = 18000
+
+const TRICASTER_WIDTH = (2 / 9) * 16
+
+export class AtemToTricasterDveConverter implements TriCasterDveConverter {
+	public convertPosition(x: number, y: number): TSR.TriCasterLayer['position'] {
+		return {
+			x: this.convertPositionX(x),
+			y: this.convertPositionY(y)
+		}
+	}
+
+	private convertPositionX(atemX: number): number {
+		const positionPercentage = atemX / ATEM_WIDTH
+		return (positionPercentage * TRICASTER_WIDTH) / 100
+	}
+
+	private convertPositionY(atemY: number): number {
+		const positionPercentage = atemY / ATEM_HEIGHT
+		return ((positionPercentage * 2) / 100) * -1
+	}
+
+	public convertScale(atemSize: number): TSR.TriCasterLayer['scale'] {
```

Comment:
Is the type here tight enough? Or should it be wrapped in a `NonNullable`, to indicate that `undefined` is never returned here?

Same goes for the `convertPosition` method.